JavaScriptSample.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <script src="https://cdn.jsdelivr.net/npm/node-forge@0.7.0/dist/forge.min.js"></script>
```

Comment:
Could you try to re-implement the example using [Subtle Crypto](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto)? The implementation is native to the browser and will not require loading additional, quite chunky, library which some partners may be iffy about.
You can see in [the supported algorithms](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto#supported_algorithms) both 'RSA-OAEP' and 'sha256' are supported.

Reply:
looking into this!

Reply:
added support for subtle crypto